src/tests/shared.rs

```diff
@@ -456,6 +452,160 @@ impl<Item> stream::Stream for Mock<Item> {
     }
 }
 
+/// Return path for the directory containing test data, like proofs and block
```

Comment:
this file is growing large.  maybe it's time to consider splitting into separate file(s).